collections.yml

```diff
@@ -17,6 +17,7 @@ Marvel Universe:
   - Captain America.*?Civil War
   - ^Ant.*?man
   - Spiderman.*?Homecoming
+  - 'Spider-Man: Homecoming'
```

Comment:
This isn't necessary since `Spiderman.*?Homecoming` works

Reply:
No, in my PMS "Spider-Man: Homecoming" didn't add it to the Marvel Universe (Spiderman.*?Homecoming). I did add "Spider-Man: Homecoming" to the file and then it added it! I think it is because of " - " .